src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -976,4 +976,45 @@
       it { expect(response).to redirect_to(project_show_path(apache_maintenance_incident_project)) }
     end
   end
+
+  describe 'POST #move_path' do
+    context 'valid project' do
+      let(:repository) { create(:repository, project: apache_project) }
+      let(:path_element) { create(:path_element, repository: repository) }
```

Comment:
does it make sense to test `move_up` and `move_down` with only one `path_element`? It will do nothing.